CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.2.0] - 2020-12-01
+### Changed
+- Set `custom.vpcDiscovery` optional. Updated readme.
```

Comment:
Probably don't need to mention that the README was updated since we should be doing that for every change